### ISSUE TITLE
Style Checker - Javascript Standard Style

### ISSUE THREAD

**Issue by ParthJinturkar** [BosamiyaSimran/SE-Project-Group-15#14]
I think we should use a tool that will automatically check the styling of the code, especially in JavaScript for better code clarity.

**Comment by thosaniparth** [comment]
Absolutely, I think that JavaScript has the option to not put a comma in the code but I believe that putting it at the appropriate location will make look code cleaner and better. I will start by putting the dependency and include it into the code and also fix it.
